pages/conduct/index.tsx

```diff
@@ -0,0 +1,37 @@
+import Head from 'next/head';
+import styles from './conduct.module.scss';
+import Page from '@layouts/page/page';
+import Header from '@components/header/header';
+import { HL } from '@components/typography/typography';
+import Stack from '@components/stack/stack';
+import Rules from '../../content/pages/code-of-conduct.mdx';
+
+const conductPage = () => (
+  <div>
+    <Head>
+      <title>Code of Conduct - UNIHACK</title>
+    </Head>
+    <Page>
+      <Header />
+      <section className={styles['conduct']}>
+        <div className={styles['conduct-content']}>
+          <Stack size="medium">
+            <HL>Code of Conduct</HL>
+            <div className={styles['introduction']}>
+              **We believe that every single person has the right to hack in a
```

Comment:
Can you wrap the `We believe that every single person has the right to hack in a safe and welcoming environment` bit inside a <strong> tag?